crates/driver/src/tests/setup/mod.rs

```diff
@@ -767,6 +772,10 @@ impl<'a> Solve<'a> {
             blockchain: self.blockchain,
         }
     }
+
+    pub fn not_ok(self, code: hyper::StatusCode) {
+        assert_eq!(self.status, code);
+    }
```

Comment:
The name `not_ok` is weird since you could also pass `200` as the expected `code`, no?
Maybe just `status()` instead?